pybamm/models/full_battery_models/lead_acid/higher_order.py

```diff
@@ -198,7 +198,7 @@ def set_full_porosity_submodel(self):
         Update porosity submodel, now that we have the spatially heterogeneous
         interfacial current densities
         """
-        self.submodels["full porosity"] = pybamm.porosity.ReactionDriven(
+        self.submodels["full porosity"] = pybamm.porosity.ReactionDrivenODE(
```

Comment:
At the moment this escapes coverage (and I think it does in the main branch too) because all the other classes overwrite it. What should I do?

Reply:
don't worry about it, it's a bug but an extremely low priority one (this is just to get the right porosity distribution in lead-acid models)